packages/components/src/local/molecules/admin-message-creator/index.tsx

```diff
@@ -30,7 +30,7 @@ export const AdminMessageCreator: React.FC<Props> = ({ from, channel, role, post
           force: from.name,
           forceColor: from.color,
           role: role,
-          icon: from.icon ? from.icon : ''
+          iconURL: from.iconURL || (from.icon || '')
```

Comment:
Aah, I think there is an issue here @sanjayradadiya 

In the legacy games I _think_ that the `icon` is the Base64 string representation of the icon itself.  But, `iconURL` is always the URL.  In the above, is there a change that we may put the Base64 icon into the `iconURL` field, and cause later problems?

Reply:
It doesn't cause any problem because it will accept base64 first time but later on when user change it icon from admin side it will be replaced  by URL so it will work on both side.

Reply:
Great, thanks.